draft-rescorla-tls-esni.md

```diff
@@ -286,16 +280,27 @@ The encrypted SNI is carried in an "encrypted_server_name"
 extension, which contains an EncryptedSNI structure:
 
 ~~~~
+   // Copied from TLS 1.3
+   struct {
```

Comment:
What is this struct used for?

Reply:
Nice catch! We no longer need it. Addressed in f55f9c3.